packages/fast-components-styles-msft/src/typography/index.ts

```diff
@@ -3,9 +3,11 @@ import { ComponentStyles } from "@microsoft/fast-jss-manager";
 import { TypographyClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 import { applyTypeRampConfig } from "../utilities/typography";
 import { neutralForegroundRest } from "../utilities/color";
+import { applyCursorDefault } from "../utilities/cursor";
 
 const styles: ComponentStyles<TypographyClassNameContract, DesignSystem> = {
     typography: {
+        ...applyCursorDefault(),
```

Comment:
We don't want the text cursor?

Reply:
Based on the though that we could use cursors for a11y cues, it seems like there is less value in having the cursor change over static text. Many web apps turn that and other things like right-click and selection off, so I considered this as our default. We should consider if that's not right though.